docs/elements/hx-pill/index.html

```diff
@@ -0,0 +1,62 @@
+---
+title: <hx-pill>
+also:
+  components/pills: Pills
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The custom <code>{{page.title}}</code> element provides a user
+      with an indication that something has been selected.
```

Comment:
Is this correct verbiage? I'm not sure every use of a pill will indicate selection. Maybe...
```
... provides a pill-like element that can be optionally dismissed.
```